tests/resources/Harbor-Pages/Project-Members.robot

```diff
@@ -21,69 +21,254 @@ ${HARBOR_VERSION}  v1.1.1
+    Sleep  2
+    Capture Page Screenshot  gointo1_${project}.png
+    # search icon
+    Click Element  xpath=/html/body/harbor-app/harbor-shell/clr-main-container/div/div/project/div/div/div[2]/div[2]/hbr-filter/span/clr-icon/svg
+    Sleep  2
+    # text search project
+    Input Text  xpath=/html/body/harbor-app/harbor-shell/clr-main-container/div/div/project/div/div/div[2]/div[2]/hbr-filter/span/input  ${project}
+    Sleep  5
+    Wait Until Page Contains  ${project}
+    Click Element  xpath=/html/body/harbor-app/harbor-shell/clr-main-container/div/div/project/div/div/list-project/clr-datagrid/div/div/div/div/div[2]/clr-dg-row/clr-dg-row-master/clr-dg-cell[2]/a
+    Sleep  3
+    Capture Page Screenshot  gointo2_${project}.png
+    	
 Add User To Project Admin
-	[Arguments]  ${project}  ${user}
-	Go Into Project2
-	Sleep  2  
-	Click Element  xpath=${project_member_tag_xpath}
-	Sleep  1
+    [Arguments]  ${project}  ${user}
+    Go Into Project2
+    Sleep  2  
+    Click Element  xpath=${project_member_tag_xpath}
+    Sleep  1
     Click Element  xpath=${project_member_add_button_xpath}
     Sleep  2	
     Input Text  xpath=${project_member_add_username_xpath}  ${user}
-	Sleep  3
+    Sleep  3
     Click Element  xpath=${project_member_add_admin_xpath}
     Click Element  xpath=${project_member_add_save_button_xpath}
-	Sleep  4
-	
+    Sleep  4
+    
 Search Project Member
-	[Arguments]  ${project}  ${user}
-	Go Into Project  ${project}
-	Sleep  2   
-	Click Element  xpath=${project_member_tag_xpath}
-	Sleep  1	
-	Click Element  xpath=${project_member_search_button_xpath}
-	Sleep  1	
-	Click Element  xpath=${project_member_search_text_xpath}
-	Sleep  2
-	Wait Until Page Contains  ${user}	
-	
+    [Arguments]  ${project}  ${user}
+    Go Into Project  ${project}
+    Sleep  2   
+    Click Element  xpath=${project_member_tag_xpath}
+    Sleep  1	
+    Click Element  xpath=${project_member_search_button_xpath}
+    Sleep  1	
+    Click Element  xpath=${project_member_search_text_xpath}
+    Sleep  2
+    Wait Until Page Contains  ${user}	
+    
 Change Project Member Role
-	[Arguments]  ${project}  ${user}  ${role}
-	Click Element  xpath=//clr-dg-cell//a[contains(.,"${project}")]
+    [Arguments]  ${project}  ${user}  ${role}
+    Click Element  xpath=//clr-dg-cell//a[contains(.,"${project}")]
     Sleep  2    
-	Click Element  xpath=${project_member_tag_xpath}
-	Sleep  1	
-	Click Element  xpath=//project-detail//clr-dg-row-master[contains(.,"${user}")]//clr-dg-action-overflow
-	Sleep  1	
+    Click Element  xpath=${project_member_tag_xpath}
+    Sleep  1	
+    Click Element  xpath=//project-detail//clr-dg-row-master[contains(.,"${user}")]//clr-dg-action-overflow
+    Sleep  1	
     Click Element  xpath=//project-detail//clr-dg-action-overflow//button[contains(.,"${role}")]
-	Sleep  2
-	Wait Until Page Contains  ${role}
-	
-	
-	
\ No newline at end of file
+    Sleep  2
+    Wait Until Page Contains  ${role}
+    
+User Can Change Role
```

Comment:
can we just separate the xpath data from function. It would be better to add them into Project_Member_Element.robot